syntax = "proto3";

package launchtor;

service TorRequester {
	rpc RequestLaunch (ConfigPorts) returns (LaunchReply) {}
}

message ConfigPorts {
	int32 socksPort = 1;
	int32 controlPort = 2;
}

message LaunchReply {
	bool success = 1;
}
